src/cli/main.rs

```diff
@@ -161,7 +161,14 @@ fn main() {
                     if raw {
                         print!("{}", response);
                     } else {
-                        print!("{:#}\n", response);
+                        if let Some(r) = response.get("result") {
+                            // If the response is null, don't print anything
+                            if Json::Null != *r {
+                                println!("{:#}", r);
+                            }
```

Comment:
Shouldn't we be printing `null` ? Anyways it's a violation of the JSONRPC2 spec to not return an object as result iirc

Reply:
If you prefer we can return `null`, I'm not really sure what's the convention here

Reply:
Yeah, let's do that. Don't know what's best either but when in doubt Explicit Is Always Better :tm: